db/migrations/20180524105631_add_name_to_movies.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.schema.table('movies', (table) => {
```

Comment:
nit: This can be collapsed into a single line, `return Knex.schema.table('movies', (table) => table.text('name'))` and the same for below as well.